module/VuFind/src/VuFind/Auth/AbstractBase.php

```diff
@@ -425,9 +460,11 @@ public function getUsernamePolicy()
     /**
      * Get password policy for a new password (e.g. minLength, maxLength)
      *
+     * @param ?string $target Authentication target for methods that support target selection
+     *
      * @return array
      */
-    public function getPasswordPolicy()
+    public function getPasswordPolicy(?string $target = null)
```

Comment:
Should this change also be made to the interface?

Reply:
Done. Added return value type as well.